lib/helpers.js

```diff
@@ -232,11 +235,36 @@ function writeSourceMaps(file, destPath, callback) {
   }
 
   // Append source map comment
-  file.contents = appendBuffer(file.contents, comment);
+  file.contents = appendBuffer(file.contents, new Buffer(comment));
```

Comment:
Should this check to ensure comment is not empty before appending?

Reply:
@sttk I removed this change. It reduced code coverage, so I investigated and found that every code path would result in a string comment being produced.